CProfiles/merge_cprofiles_parallel.py

```diff
@@ -71,11 +71,23 @@ def main():
     parser.add_argument("-v", "--verbose", action="store_true")
 
     parser.add_argument( "indir", type=str, nargs=1, help = "directory containing produced files")
+    parser.add_argument( "pid", type=str, help = "particle type (eg e-, mu+...", default="all")
```

Comment:
indent?

Reply:
fixed? I am not using indentation in pycharm so maybe it failed?